source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -0,0 +1,250 @@
+===================================
+Document Data Format: Extended JSON
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Overview
+--------
+
+JSON is a data format that represents data as objects that contain one or
```

Comment:
This doesn't seem 100% technically accurate, since JSON doesn't have to encode an object and objects can be empty. Maybe to sharpen focus on the point you're making:

>JSON is a data format that encodes the values of objects, arrays, numbers, and so on. Extended JSON adds special syntax to encode type information for those values...